cluster/gce/util.sh

```diff
@@ -2571,7 +2576,7 @@ function create-master() {
     --allow tcp:443 &
 
   echo "Configuring firewall for apiserver konnectivity server"
-  if [[ "${ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "true" ]]; then
+  if [[ "${EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "true" ]]; then
```

Comment:
It seems like this determines if the agents can connect to the server. I believe we want this turned on, otherwise the agents will behave badly (crashloop?)

Reply:
You mean moving this to `RUN_KONNECTIVITY_SERVICE` instead of keeping it at `ENABLE_EGRESS_VIA_KONNECTIVITY`?